dt/bmo01/dataplane/baremetalhosts/baremetalhost_template.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  labels: {}
+  name: _ignored_
+  namespace: openstack
+  inspect.metal3.io: _replaced_
```

Comment:
```suggestion
  annotations:
    inspect.metal3.io: _replaced_
```

Reply:
ah, thanks! 

Better now?
It "renders":

```
---
apiVersion: metal3.io/v1alpha1
kind: BareMetalHost
metadata:
  annotations:
    inspect.metal3.io: disabled
  labels:
    app: openstack

```

Reply:
Yep, that looks good :+1: